src/models/propulsion/FGTurbine.h

```diff
@@ -337,6 +341,34 @@ class FGSpoolUp : public FGParameter
   FGTurbine* turb;
   double delay; ///< Inverse spool-up time from idle to 100% (seconds)
 };
+
+class FGTSFC : public FGParameter
+{
+public:
+  FGTSFC(FGTurbine* _turb, double tsfcVal)
+    : turb(_turb), tsfc(new FGRealValue(tsfcVal)) {}
```

Comment:
This class just need to manage the case where `<tsfc>` is specified as a number. So you don't need `FGRealValue`.
```suggestion
  FGTSFC(FGTurbine* _turb, double tsfcVal)
    : turb(_turb), tsfc(tsfcVal) {}
```